README.md

```diff
@@ -172,6 +172,21 @@ repeating the mistakes of the past.
     - For services, checkout our [best practices](https://mobify.atlassian.net/wiki/display/SYS/Service+Best+Practices).
     - For front-end web apps, checkout our [best practices](https://mobify.atlassian.net/wiki/questions/85917798/what-are-our-best-practices-for-building-front-end-web-application).
 
+### Use checklists
+
+There is [plenty developers can learn from pilots](http://lanyrd.com/2015/pycon/sdmfwy/#link-cbgfg) - one is that our memories are faulty and unreliable, leading to inconsistency.
+
+Checklists are a great way to ensure consistency, in areas such as:
+
+- Code changes following best practices
```

Comment:
@jansepar I feel like its possible to link to direct examples of these being used in the wild.